db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20190611212339) do
+ActiveRecord::Schema.define(version: 20200423205608) do
```

Comment:
We prefer not to update the schema file in the same pull as migrations -- instead, we'll run the migration on production and get a schema dump from there.